alpha_automl/builtin_primitives/image_encoder.py

```diff
@@ -32,7 +32,11 @@ def transform(self, images):
         data = []
         if isinstance(images, pd.DataFrame):
             for file in images[images.columns[0]]:
-                im = imread(file)
+                try:
+                    im = imread(file)
+                except:
+                    logger.error(f"[FATAL]The dataset contains invalid image file, please remove them before training: {file}")
```

Comment:
We need to capture the actual exception and give more details about the actual error to the log message. Maybe something like:

"Using a zeros vector instead of the actual image due to failure while reading the image at path [{image-path}]. Please fix this image or remove it from the training data. Loading failed due to error: {exception error + stacktrace}."

Also, the term FATAL should be used only if we actually kill/stop the pipeline execution or search. Otherwise, it may be misleading.

Reply:
Fixed.